src/uct/ib/ud/base/ud_iface.c

```diff
@@ -68,11 +71,20 @@ uct_ud_iface_cep_get_conn_sn(uct_ud_iface_t *iface,
                              int path_index)
 {
     void *peer_address = ucs_alloca(iface->conn_match_ctx.address_length);
-    return (uct_ud_ep_conn_sn_t)
-           ucs_conn_match_get_next_sn(&iface->conn_match_ctx,
-                                      uct_ud_iface_cep_get_peer_address(
-                                          iface, ib_addr, if_addr, path_index,
-                                          peer_address));
+    uct_ud_ep_conn_sn_t conn_sn;
+
+    conn_sn =
+            (uct_ud_ep_conn_sn_t)
```

Comment:
why need explicit cast to uct_ud_ep_conn_sn_t?

Reply:
`ucs_conn_sn_t` is 64-bit value, `uct_ud_ep_conn_sn_t` is 32-bit value

Reply:
I think explicit cast not needed here, could simplify the code